.github/workflows/ci.yml

```
@@ -90,3 +90,17 @@ jobs:
         path: |
           dist/ffmuc-meet.exe
           dist/latest.yml
+
+  winget-release:
+    name: Publish to WinGet
+    runs-on: ubuntu-latest
+    needs: build-windows
+    if: github.event_name == 'release' && github.event.action == 'published'
+    steps:
+    - uses: actions/checkout@v4
+    - name: Publish WinGet Manifest
+      uses: vedantmgoyal9/winget-releaser@v2
+      with:
+        identifier: freifunkMUC.FreifunkMeet
```

maybe we should add a filter same as in this example: https://github.com/vedantmgoyal9/winget-releaser#examples-

so we only use .exe 

```suggestion
        identifier: freifunkMUC.FreifunkMeet
        installers-regex: '\.exe$' # Only .exe files
```